include/operations/blas2_trees.hpp

```diff
@@ -318,7 +319,7 @@ struct RedRowMatVct {
     }
     return valWG;
   }
-#endif // ORIGINAL_CODE
+#endif  // ORIGINAL_CODE
```

Comment:
This is for blas2, and we have not really gone into that yet. I'd better leave as it is and deal with it in another pull request.